tools/restatectl/src/commands/log/reconfigure.rs

```diff
@@ -95,7 +96,34 @@ async fn reconfigure(connection: &ConnectionInfo, opts: &ReconfigureOpts) -> any
             segment_index: opts.segment_index,
             params,
         })
-        .await?;
+        .await?
+        .into_inner();
+
+    c_println!("✅ Log reconfiguration");
+    c_println!(" └ Segment Index: {}", response.new_segment_index);
+    let Some(sealed_segment) = response.sealed_segment else {
+        return Ok(());
```

Comment:
I am not sure if this can actually happen at the moment but maybe the server returns `None` as a form of "this was a no-op" - maybe we should print something here for the sake of completeness on the client-side? Feel free to disregard.

Reply:
the server will ALWAYS return `Som(sealed_segment)` otherwise, it will return an error. the Option is added by protobuf. This could be an `unwrap` but decided to gracefully handle it to avoid possible comparability issue in the future